src/components/20-molecules/accordion/index.js

```diff
@@ -46,62 +49,66 @@ class AXAAccordeon extends LitElement {
 
   firstUpdated() {
     this.getFirstChild().shadowRoot.querySelector(
-      '.m-accordeon__container'
+      '.m-accordion__container'
     ).style.borderTop = 'solid 1px #ccc';
 
-    // slot-content is not available on IE 11 at this time
-    if (window.document.documentMode) {
-      setTimeout(() => {
-        this.toggleAnimation();
-      });
-    } else {
-      this.toggleAnimation();
-    }
+    window.addEventListener('resize', this.recalculateDimensionsOnResize);
+
+    this.toggleAnimation();
   }
 
   toggleAnimation() {
-    const accordeonContent = this.shadowRoot.querySelector(
-      '.m-accordeon__content'
+    const accordionContent = this.shadowRoot.querySelector(
+      '.m-accordion__content'
     );
 
+    if (!accordionContent) {
+      return;
+    }
+
     if (this.open) {
```

Comment:
While youre' at it, you might as well DRYify it more here:

`accordionContent.style.maxHeight = this.open ? `${accordionContent.scrollHeight}px` : 0;`